src/quo2/foundations/colors.cljs

```diff
@@ -256,9 +256,9 @@
             color-keyword (keyword color)
             base-color    (get-in colors-map
                                   [color-keyword suffix])]
-        (if hex?
-          color
-          (if opacity (alpha base-color (/ opacity 100)) base-color)))))))
+        (if opacity
+          (alpha (if hex? color base-color) (/ opacity 100))
+          (if hex? color base-color)))))))
```

Comment:
@J-Son89 I have made a small change here to handle opacity for hexa values. Please have a look.

Reply:
yeah looks good, perhaps we can use `cond` to flatten the structure?

Reply:
Not sure how `cond` will improve this!

Reply:
imo it removes the nested if's and becomes more obvious of what is being set and when. e.g something like:
```
(cond
  (and opacity hex) (alpha color (/ opacity 100))
  opacity (alpha base-color (/ opacity 100))
  hex? color
  :else base-color)
```

Reply:
Done